website_product_document_download_counter/tests/product_document_test_utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Cetmix OÜ
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL).
+
+import base64
+
+
+class ProductDocumentTestUtils:
+    """ """
+
+    def _create_product_with_document(self, product_name="Test Product"):
+        product = self.env["product.template"].create(
+            {
+                "name": product_name,
+            }
+        )
```

Comment:
Removing the trailing comma, you have an auto-formatter friendlier with the number of lines:

```suggestion
        product = self.env["product.template"].create(
            {"name": product_name
        )
```